src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_VALUE_SIZE = 10;
+    private Object[] keys = new Object[MAX_VALUE_SIZE];
+    private Object[] values = new Object[MAX_VALUE_SIZE];
+    private int count = 0;
```

Comment:
```suggestion
    private int count;
```

we dont need to initialize int values bu 0 as it will be initialized by 0 by default